### PR TITLE
Update testing.md

### LINKED ISSUE #101: Run spellchecking and improve working in testing doc
* Run `aspell` or a similar tool
* Improve wording. A few examples: `come in candy`, `cmoputer`

### PR DESCRIPTION
**Pull request by Rhantolq**
References #101

### CODE REVIEW

**Comment by kantoniak** [comment]
I made a few changes:
* Fixed some puctuation
* Made some wording changes
* Rewrapped all text at column 80
* Updated version number

@avan1235 @Rhantolq can one of you read again and (un)approve?

File: specs/testing.md [hunk 1]
```diff
@@ -1,177 +1,202 @@
...
+definitions of testing parts in GitHub workflows that builds the debug and
+release versions of application. They would run as Gradle tasks in a
+pre-defined process which publish artifacts to GitHub store when successfully
+completed.
 
 #### Camera devices
 
-Camera device code is covered by `pytest` unit tests. These tests can be run twofold:
+Parts of the camera code can be run and tested in any environment. This part is
+covered by `pytest` unit tests. These tests can be run twofold:
```
Comment (Rhantolq) [inline specs/testing.md:59]: maybe change twofold to _twofold_

File: specs/testing.md [hunk 2]
```diff
@@ -1,177 +1,202 @@
...
 
 ### Unit testing
 
 #### Django server
 
-Django [unit tests](https://docs.djangoproject.com/en/3.0/topics/testing/overview/) allow us to test many aspects of the server:
-* Database tests, assuring that the database follows the designed model, works well under load and is 
-properly configured. Also some rollback emulation testing
-* Normal unit tests for testing the server app against code bugs.
+Django [unit
```
Comment (Rhantolq) [inline specs/testing.md:24]: Weird \n

File: specs/testing.md [hunk 3]
```diff
@@ -1,177 +1,202 @@
...
+tests](https://docs.djangoproject.com/en/3.0/topics/testing/overview/) allow us
+to test many aspects of the server:
+* Database tests, assuring that the database follows the designed model, works
+  well under load and is properly configured. Database tests execute some
+  rollback emulation testing.
+* Regular unit tests for testing the server app against regressions.
 
 #### Android application
```
Comment (Rhantolq) [inline specs/testing.md:32]: Read for @avan1235

Comment (Rhantolq) [reply specs/testing.md:32]: I,m leaving you all ### Android

**Pull request merged**